### ISSUE TITLE
Issues when deleting a list item on Android

### ISSUE THREAD

**Issue by ognsya** [siyuan-note/siyuan#8745]
![screen-20230712-210042_2](https://github.com/siyuan-note/siyuan/assets/67518000/5ae0f192-693a-4715-b085-dae363310cc3)
### Is there an existing issue for this?

- [X] I have searched the existing issues

### Can the issue be reproduced with the default theme (daylight/midnight)?

- [X] I was able to reproduce the issue with the default theme

### Could the issue be due to extensions?

- [X] I've ruled out the possibility that the extension is causing the problem.

### Describe the problem

When deleting the text in a list item, as soon as the last character is deleted, the on-screen keyboard closes. I expect the keyboard to remain.

After that, if I tap to place the cursor back in the (now empty) list item, and press Delete,  only the bullet is deleted. I expect the list item to be removed (and the cursor to be placed at the end of the list item above). Also, the keyboard goes away once again - I expect it to remain.

### Expected result

I describe it above.

### Version environment

```markdown
- Version: 2.9.4
- Operating System:  Android
- Browser (if used):
```

### Log file

I don't know how to find that on my phone.

**Comment by ognsya** [comment]
I added a GIF above showing the described behaviour.

**Comment by 88250** [comment]
Hello, have you used an input method other than the system's own?

**Comment by ognsya** [comment]
> Hello, have you used an input method other than the system's own?

I don't know any other input method... Do you mean an external keyboard? I don't have that.

**Comment by 88250** [comment]
Please export the system log, in the right panel About - Export

**Comment by ognsya** [comment]
> Please export the system log, in the right panel About - Export

Does this not happen if you test it on mobile?
My phone is a Pixel 6 by the way.

Here's my log:

  ____    _  __   __
 / ___|  (_) \ \ / /  _   _    __ _   _ __
 \___ \  | |  \ V /  | | | |  / _` | | '_ \
  ___) | | |   | |   | |_| | | (_| | | | | |
 |____/  |_|   |_|    \__,_|  \__,_| |_| |_|
I 2023/07/13 03:09:40 runtime.go:74: kernel is booting:
    * ver [2.9.5]
    * arch [arm64]
    * os [Android 13/SDK 33/WebView 114/Manufacturer Google/Brand google]
    * pid [26469]
    * runtime mode [prod]
    * working directory [/data/user/0/org.b3log.siyuan/files/app/app]
    * read only [false]
    * container [android]
    * database [ver=20220501]
    * workspace directory [/storage/emulated/0/Android/data/org.b3log.siyuan/files/siyuan]
I 2023/07/13 03:09:40 conf.go:110: initialized the specified language [en_US]
I 2023/07/13 03:09:40 conf.go:232: upgraded from version [2.9.4] to [2.9.5]
I 2023/07/13 03:09:40 runtime.go:128: use network proxy [system]
I 2023/07/13 03:09:40 serve.go:114: kernel [pid=26469] http server [0.0.0.0:6806] is booting
I 2023/07/13 03:09:40 blocktree.go:446: read block tree [659 kB] to [/storage/emulated/0/Android/data/org.b3log.siyuan/files/siyuan/temp/blocktree], elapsed [0.04s]
I 2023/07/13 03:09:40 conf.go:710: database size [7.9 MB], tree/block count [64/2386]
I 2023/07/13 03:09:40 working.go:153: kernel booted
I 2023/07/13 03:09:40 box.go:79: auto stat [trees=8, blocks=349, dataSize=2.1 MB, assetsSize=965 kB]
I 2023/07/13 03:09:40 index.go:133: rebuilt database for notebook [20210808180117-6v0mkxr] in [0.03s], tree [count=56, size=1.1 MB]
I 2023/07/13 06:54:23 box.go:79: auto stat [trees=64, blocks=2390, dataSize=3.3 MB, assetsSize=2.0 MB]
I 2023/07/13 08:17:59 index.go:133: rebuilt database for notebook [20210808180117-6v0mkxr] in [0.09s], tree [count=56, size=1.1 MB]
I 2023/07/13 08:17:59 index.go:206: resolved refs [27] in [119ms]
I 2023/07/13 10:15:57 box.go:79: auto stat [trees=63, blocks=2364, dataSize=3.3 MB, assetsSize=2.0 MB]
E 2023/07/13 13:15:24 transaction.go:882: load tree [20230713131453-4h7lrhk] failed: block not found
I 2023/07/13 13:15:28 database.go:108: reinitialized database [/storage/emulated/0/Android/data/org.b3log.siyuan/files/siyuan/temp/siyuan.db]
I 2023/07/13 13:15:28 index.go:133: rebuilt database for notebook [20230706094229-4rj8q3f] in [0.02s], tree [count=7, size=133 kB]
I 2023/07/13 13:15:28 index.go:133: rebuilt database for notebook [20210808180117-6v0mkxr] in [0.08s], tree [count=56, size=1.1 MB]
I 2023/07/13 13:15:51 index.go:206: resolved refs [27] in [112ms]
I 2023/07/13 17:01:31 box.go:79: auto stat [trees=63, blocks=2349, dataSize=3.3 MB, assetsSize=2.0 MB]
I 2023/07/13 20:25:22 box.go:79: auto stat [trees=63, blocks=2351, dataSize=3.3 MB, assetsSize=2.0 MB]
I 2023/07/13 22:55:20 box.go:79: auto stat [trees=63, blocks=2351, dataSize=3.3 MB, assetsSize=2.0 MB]
I 2023/07/14 09:28:24 box.go:79: auto stat [trees=64, blocks=2353, dataSize=3.3 MB, assetsSize=2.0 MB]
I 2023/07/14 16:32:20 box.go:79: auto stat [trees=64, blocks=2353, dataSize=3.3 MB, assetsSize=2.0 MB]
I 2023/07/14 22:10:11 box.go:79: auto stat [trees=64, blocks=2353, dataSize=3.3 MB, assetsSize=2.0 MB]
I 2023/07/15 00:29:19 box.go:79: auto stat [trees=64, blocks=2353, dataSize=3.3 MB, assetsSize=2.0 MB]
I 2023/07/15 08:49:51 box.go:79: auto stat [trees=64, blocks=2353, dataSize=3.3 MB, assetsSize=2.0 MB]

**Comment by Vanessa219** [comment]
Because we don't have this device, please try again in the next version, thank you.

**Comment by ognsya** [comment]
> Because we don't have this device, please try again in the next version, thank you.

Seems fixed in 2.9.7!

Thanks V 👍
